sphinx/ext/cylc-lang.py

```diff
@@ -0,0 +1,261 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# (C) British Crown Copyright 2012-7 Met Office.
+#
+# This file is part of Rose, a framework for meteorological suites.
+#
+# Rose is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Rose is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Rose. If not, see <http://www.gnu.org/licenses/>.
+# -----------------------------------------------------------------------------
+"""An extension providing pygments lexers for suite.rc files and cylc graph
+strings."""
+
+from pygments.lexer import RegexLexer, bygroups, include, words
+from pygments.token import *
```

Comment:
Agreed, It's certainly in pylint, don't know about pep8. I'd left this has a star import as there are quite a lot of namespaces to import!